apps/system/style/app_titlebar.css

```diff
@@ -26,12 +27,32 @@
   height: .2rem;
 }
 
-.appWindow .titlebar .titlebar-maximized {
+/* Statusbar shadow hidden by default */
+.appWindow .titlebar {
+  visibility: hidden;
+}
+
+/* Statusbar shadow visible on app transitions */
+.appWindow.transition-opening > .titlebar,
+.appWindow.transition-opening > .chrome.maximized + .titlebar > .titlebar-maximized,
+.appWindow.transition-opening > .chrome+ .titlebar > .titlebar-minimized,
+.appWindow.transition-closing > .chrome.maximized + .titlebar > .titlebar-maximized,
+.appWindow.transition-closing > .chrome+ .titlebar > .titlebar-minimized,
+/* Statusbar shadow visible on edge gestures */
+.appWindow.inside-edges > .titlebar,
+.appWindow.outside-edges-right > .titlebar,
+.appWindow.outside-edges-left > .titlebar,
```

Comment:
you want to add `.appWindow.will-become-active > .titlebar,` otherwise there's a flash of the icon while we remove the edge overlay